lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -356,7 +356,7 @@ static StringRef getVerilogDeclWord(Operation *op,
   }
   if (isa<WireOp>(op))
     return "wire";
-  if (isa<ConstantOp, LocalParamOp, ParamValueOp>(op))
+  if (isa<ConstantOp, LocalParamOp, ParamValueOp, FileDescriptorOp>(op))
```

Comment:
Is this necessary given that this should always inline?  Is there a way to hit this in a test?

Reply:
It's not hit in any tests, the constant seems to be always inlined. I thought in case that does not happen, emitting a localparam might have been the sensible thing. Maybe I could add an assertion to ensure that we never try to declare something initialised with a FileDescriptorOp?